nbstripout/_utils.py

```diff
@@ -94,6 +96,34 @@ def strip_zeppelin_output(nb: dict) -> dict:
             cell['results'] = {}
     return nb
 
+class OutputType:
```

Comment:
I probably wouldn't make this a class at all and simply add `matches_output` as a free standing function doing the validation logic. IMO this is more complex than it needs to be, but I won't insist if you'd rather keep this as a class 😃